brave/src/main/java/brave/internal/baggage/BaggageHandler.java

```diff
@@ -70,22 +70,22 @@
   @Nullable S updateState(S state, BaggageField field, @Nullable String value);
 
   /**
-   * Extracts any state from a remote value received by {@link Propagation.Getter#get(Object,
+   * Extracts any state from a request value received by {@link Propagation.Getter#get(Object,
    * Object)}.
    *
-   * <p>Ex. When the state is a simple string, this will just use the remote value directly.
+   * <p>Ex. When the state is a simple string, this will just use the request value directly.
    * {@linkplain #isDynamic() Dynamic values} will need to perform some decoding, such as splitting
    * on comma and equals.
    */
-  @Nullable S fromRemoteValue(String remoteValue);
+  @Nullable S fromRequestValue(Object request, String value);
```

Comment:
we aren't currently using this, but we will need this for secondary sampling. This was what I was changing and realized.. wow, we should really stop using so many weird terms!